pkg/app/handlers/users/handler.go

```diff
@@ -227,3 +234,35 @@ func (u UsersResourceHandler) setUserMappings(ctx context.Context, userID string
 	}
 	return nil
 }
+
+func (u UsersResourceHandler) getIdentityRelation(userID, identity string) *dsc.Relation {
+	object, relation, found := strings.Cut(u.cfg.SCIM.IdentityRelation, "#")
+	if found {
+		switch object {
+		case u.cfg.SCIM.UserObjectType:
+			return &dsc.Relation{
+				SubjectId:   userID,
+				SubjectType: u.cfg.SCIM.UserObjectType,
+				Relation:    relation,
+				ObjectType:  u.cfg.SCIM.IdentityObjectType,
+				ObjectId:    identity,
+			}
+		case u.cfg.SCIM.IdentityObjectType:
+			return &dsc.Relation{
+				SubjectId:   identity,
+				SubjectType: u.cfg.SCIM.IdentityObjectType,
+				Relation:    relation,
+				ObjectType:  u.cfg.SCIM.UserObjectType,
+				ObjectId:    userID,
+			}
+		}
+	}
+
+	return &dsc.Relation{
+		SubjectId:   userID,
+		SubjectType: u.cfg.SCIM.UserObjectType,
+		Relation:    u.cfg.SCIM.IdentityRelation,
+		ObjectType:  u.cfg.SCIM.IdentityObjectType,
+		ObjectId:    identity,
```

Comment:
I think we should default to `user#identifier`. This service is primarily used to populate the self-hosted controller, which uses the inverted identity relation. And regardless, we're moving all our samples to have identities as the subject. That should be the default behavior.